.github/workflows/terraform.yaml

```diff
@@ -0,0 +1,30 @@
+name: Check the Terraform Modules
```

Comment:
I think this workflow has already been moved to the `sdcore-github-workflows` project, right?

Reply:
yes, it's been removed. This repo is not updated recently. I will update it.

Reply:
Done